.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Shouldn't the whole .vscode/ folder be added to the .gitignore?

Reply:
Not since VS Code keeps certain editor configs in there. For example, to get VS Code to reliably indent with tabs instead of spaces, `editor.insertSpaces` must be set to `false` for the project. I've found waiting for ESLint to enforce that on save to be annoying, so I prefer this route. The `extensions.json` file informs VS Code to ask users to install certain plugins, which I think would make the development experience easier for all of us. We can bikeshed the extension list, but to start, I've added the Prettier, ESLint, DOTENV, Prisma, and TODO-Tree plugins, since these relate directly to the dev environment I'm suggesting here.

I'd happily commit other editor workspace configs as well, if they make sense to do so.